packages/ui/src/Field.tsx

```diff
@@ -186,24 +187,19 @@ export const Field = ({type, ...rest}: FieldProps) => {
     //   </>
     // );
   } else if (type === "customSelect") {
-    return null;
-    // if (!options) {
-    //   console.error("Field with type=customSelect require options");
-    //   return null;
-    // }
-    // return (
-    //   <CustomSelect
-    //     {...rest}
-    //     disabled={disabled}
-    //     options={options}
-    //     placeholder={placeholder}
-    //     value={value}
-    //     onChange={(val) => {
-    //       onChange(val);
-    //       onBlur && onBlur(val);
-    //     }}
-    //   />
-    // );
+    if (!rest?.options) {
+      console.error("Field with type=customSelect require options");
+      return null;
+    }
+    return (
+      <CustomSelectField
+        {...(rest as CustomSelectFieldProps)}
+        onChange={(result) => {
+          rest.onChange(result);
+          rest.onBlur && rest.onBlur(result);
```

Comment:
See thread between myself and @kjkurtz in dev about this. We will need to add onblur back in in the Field component where it was removed for other types of field, and/or move this down into the individual components. But it looks like we removed onBlur functionality in some of these during this re-working.